code/modules/clothing/spacesuits/rig/modules/combat.dm

```diff
@@ -246,7 +246,8 @@
 
 /obj/item/rig_module/mounted/energy_blade/activate()
 
-	..()
+	if(!..())
```

Comment:
use .=..() unless you have a specific reason to do otherwise

and don't return 0 specifically unless theres a similar reason, you're overriding the parent return

Reply:
..() checks if we have enough power and returns TRUE or FALSE. That's the reason I used if(!..())